cypress/e2e/demoblaze.cy.ts

```diff
@@ -0,0 +1,70 @@
+import {LoginPage, ShoppingPage} from '../pages/index'
+
+const loginPage = new LoginPage();
+const shoppingPage = new ShoppingPage();
+
+const products = Cypress.env("Products");
+const placeOrder = Cypress.env("PlaceOrder");
+const contactInfo = Cypress.env("ContactInfo");
+
+const phoneElement = products.phoneElement;
+const laptopElement = products.laptopElement;
+const monitorElement = products.monitorElement;
+
+describe("Log in feature", () => {
+  beforeEach(() => {
+    cy.clearAllCookies()
+    cy.visit("https://www.demoblaze.com/index.html");
+    loginPage.verfifylogInOption();
+  });
+
+  it("Add products to cart and verify Title Product", () => {
+    shoppingPage.addElementToCart(phoneElement);
+    cy.get(".btn-lg").contains("Add to cart").click();
```

Comment:
Este test está muy bien! Así está claro y secuencial.

 Te recomendaría (si quieres hacerlo) pasar cada click a un método como lo hicimos en clase.

Que me reciba un texto, y el método busque ese texto en un botón y le de click()

Y lo puedes crear en una clase BasePage y extiendes esa clase en shoppingPage... con eso lo podrías llamar desde el test y te quedaría así:

```suggestion
    shoppingPage.clickBtnByText('Add to cart');
```

Y en BasePage sería algo como el que hicimos en el video:

`    clickBtnByText(text) {
        cy.contains(text).click();
    }`

Reply:
ok profe